threatmodel.md

```diff
@@ -10,3 +10,10 @@ It is assumed that an attacker may perform one or more the following actions:
 
 While it is not always possible to protect against all scenarios, the system should to the extent possible mitigate and/or reduce the damage caused by a successful attack, detect the occurrence of an attack and notify appropriate parties, yet remain usable for parties operating the system.  Furthermore, the system should recover from successful attacks in a way that presents low operational overhead and risk to users.
 
+Attacker Goals:
+1. Trying to have a party install a malicious image under the attackers control.
+2. Trying to have a party install an outdated image.  For example, one with known security vulnerabilities.
```

Comment:
```suggestion
2. To have a party deploy an artifact that does not have a currently valid signature.  For example, a previous version of an artifact with a signature that was revoked by the signer.
```

Reply:
This changes the meaning a bit.  The prior also would include artifacts that were not the latest (when the latest is requested), but this seems to imply those are fine.

Reply:
I updated my change. In some cases, someone might pull an old image by digest on purpose, and as long as the signature is still valid, this should be accepted, which is the subtlety I was trying to add. But yes, it still needs to use the latest tag mapping when tags are used.